i18n/locales/en.json

```diff
@@ -35,5 +35,6 @@
   "components.pagination.next": "Newer | ",
   "components.pagination.previous": "Older",
   "layouts.blogPost.author.byLine": "{author, select, null {} other {By {author}, }}",
-  "layouts.blogIndex.currentYear": "News from {year}"
+  "layouts.blogIndex.currentYear": "News from {year}",
+  "components.codeBox.copy": "{copied, select, true {copied}other {copy}}"
```

Comment:
Please remove the translation key from all languages except `en.json`. Thank you!

Also, the translation key should be `components.common.shellBox.copy` to actually properly refer to its canonical path.